legion/engine/core/ecs/registry.hpp

```diff
@@ -205,6 +204,8 @@ namespace legion::core::ecs
         //template<typename component_type>
         //static component_type& createComponent(entity target, serialization::component_prototype<component_type>&& prototype);
 
+        static void* createComponent(id_type typeId, entity target, const void* component);
```

Comment:
no documentation and a commented out older version of the function? Clean up commented out code if unused

Reply:
yeah i haven't touched anything outside of the serialization framework yet, since better ECS integration, scheduling integration, unit testing, and docs are planned for later subfeatures